src/mapillary_client.py

```diff
@@ -0,0 +1,71 @@
+from os import getenv, getcwd
+import logging
+from multiprocessing import Pool
+from pathlib import Path
+from mapillary import interface as mly
+from requests import get, RequestException
+
+
+class MapillaryClient:
+    def __init__(self, access_token: str, log_level: int = logging.INFO, basepath: str = getcwd()) -> None:
```

Comment:
We're using a convention of saving data to `data/raw` so consider having a default base directory like `data/raw/mapillary/`

https://github.com/AmericanRedCross/street-view-green-view?tab=readme-ov-file#project-organization

Reply:
I have implemented the option to set a `basepath`, so this should be possible by modifying the default `basepath`